default/stringtables/en.txt

```diff
@@ -5266,6 +5266,32 @@ Repairs are delayed for one turn for ships just entering the system. Ships need
 
 Any combat in the system will disrupt repairs for that turn.'''
 
+CONFIG_GUIDE_TITLE
+Configuration Guide
+
+CONFIG_GUIDE_TEXT
+'''Most configuration options for the game are stored and read from a single XML formatted file named config.xml.
+If this file does not exist (or is for a different build or version) it will be (re)created when the client starts. 
+When an option is changed, the configuration file is updated.
+Some of these options can be changed from the [[OPTIONS_TITLE]] menu, under the Main Menu.
+
+An option set in the persistent config file (if one exists) will override the value from config.xml.
+An option set from a command line argument will override the value from both files.
+
+
+Persistent Config File:
+
+A persistent configuration file is an optional XML file named persistent_config.xml.
+If this file exists, any settings it contains will override those from the main configuration file.
+This may be useful when a different build of the game is launched, such as after updating to a new version. To help retain compatibility, it is preferred to keep as few options in this file as needed.
+
+A persistent configuration file can be created or updated on the [[OPTIONS_TITLE]] menu [[OPTIONS_PAGE_MISC]] tab by clicking the [[OPTIONS_CREATE_PERSISTENT_CONFIG]] button.
+When using this button, a known issue is that some options may be included even though they were not changed. After updating to a new version, it may be required to manually remove these options from the file to update their default setting.
+The persistent configuration file may be manually created if desired, but should never contain the "version-string" option.
+
+The configuration files are located at: <browsepath PATH_CONFIG></browsepath>
```

Comment:
I've added `blocked by` to the PR description, will require rebase if/when it is merged.